apps/etterlatte-saksbehandling-ui/client/src/components/person/uhaandtereHendelser/HistoriskeHendelser.tsx

```diff
@@ -48,7 +54,11 @@ const HistoriskeHendelser = (props: Props) => {
                     <Table.DataCell>{type}</Table.DataCell>
                     <Table.DataCell>{gjelderPerson}</Table.DataCell>
                     <Table.DataCell>{formaterStringDato(opprettet)}</Table.DataCell>
-                    <Table.DataCell>{kommentar ? <p>{kommentar}</p> : <p>Ingen kommentar</p>}</Table.DataCell>
+                    <Table.DataCell>
+                      {type !== GrunnlagsendringsType.INSTITUSJONSOPPHOLD &&
+                        (kommentar ? <p>{kommentar}</p> : <p>Ingen kommentar</p>)}
+                      {type === GrunnlagsendringsType.INSTITUSJONSOPPHOLD && <InstitusjonsoppholdBegrunnelse id={id} />}
```

Comment:
Så kommentar er ikkje relevant viss det er institusjonsopphold? For da inngår det som elles ville ha vore i kommentaren i begrunnelse eller noko for oppholdet?

Reply:
Det er tanken!

Reply:
På sikt vil det også kanskje komme slikt for andre hendelsestyper og.